ibis/pandas/core.py

```diff
@@ -152,6 +149,14 @@ def execute_with_scope(expr, scope, aggcontext=None, clients=None, **kwargs):
     scope : collections.Mapping
         A dictionary mapping :class:`~ibis.expr.operations.Node` subclass
         instances to concrete data such as a pandas DataFrame.
+    timecontext : Tuple[pd.Timestamp, pd.Timestamp]
```

Comment:
Can you type this to be `Optional[Tuple[pd.Timestamp, pd.Timestamp]]`

Reply:
Yes that makes sense